src/coreclr/nativeaot/BuildIntegration/Microsoft.DotNet.ILCompiler.SingleEntry.targets

```diff
@@ -4,6 +4,8 @@
     <!-- Define the name of the runtime specific compiler package to import -->
     <OSIdentifier Condition="$(RuntimeIdentifier.StartsWith('win'))">win</OSIdentifier>
     <OSIdentifier Condition="$(RuntimeIdentifier.StartsWith('osx'))">osx</OSIdentifier>
+    <OSIdentifier Condition="$(RuntimeIdentifier.StartsWith('ios'))">ios</OSIdentifier>
+    <OSIdentifier Condition="$(RuntimeIdentifier.StartsWith('iossimulator'))">iossimulator</OSIdentifier>
```

Comment:
nit: we implicitly rely on the order of the checks here (e.g. if we checked `iossimulator` first and then `ios` it'd always result in iOS since both start with the same prefix).

It might be cleaner to add `Condition="'$(OSIdentifier)' == ''"` to all of these checks so that the first match wins.

Reply:
> It might be cleaner to add Condition="'$(OSIdentifier)' == ''" to all of these checks so that the first match wins.

... and flip the order of the lines then.